recipe/patches/0010-Empty-install-requirements-installed-by-conda-build.patch

```diff
@@ -40,20 +41,19 @@ index f29fd74df..3c42da44c 100644
 -    "opencensus",
 -    "prometheus_client >= 0.7.1",
 -]
--
-+install_requires = [] # everything should be installed via conda build recipe
++install_requires = []
+ 
  
  def is_native_windows_or_msys():
-     """Check to see if we are running on native Windows,
-@@ -507,7 +482,7 @@ setuptools.setup(
+@@ -520,7 +495,7 @@ setuptools.setup(
      # The BinaryDistribution argument triggers build_ext.
      distclass=BinaryDistribution,
      install_requires=install_requires,
--    setup_requires=["cython >= 0.29.14", "wheel"],
+-    setup_requires=["cython >= 0.29.15", "wheel"],
```

Comment:
this implies an update to `meta.yaml` is needed, too

Reply:
Similarly here; strictly speaking you're correct, but we already only pin `cython >=0.29`, which is more than enough (also 0.29.15 is ancient)

Reply:
If so, we should pin to `>=0.29.15` anyway (or maybe something more recent) just to be 100% sure (instead of 99.9% as is now 😆)